### ISSUE TITLE
[469b] Speed setup description mislead users nowadays

### ISSUE THREAD

**Issue by SeriousBuggie** [OldUnreal/UnrealTournamentPatches#880]
![img](https://user-images.githubusercontent.com/70026933/174469469-64904544-0e92-4ad6-a546-0376803c180d.png)

Players often use this combo as it described. but ISDN in modern years much better from LAN from 2000 year.
UMenuNetworkClientWindow:
```js
		case 0:
			NewSpeed = 2600;
			break;
		case 1:
			NewSpeed = 5000;
			break;
		case 2:
			NewSpeed = 10000;
			break;
		case 3:
			NewSpeed = 20000;
			break;
```
```js
	NetSpeedText="Internet Connection"
	NetSpeedHelp="Select the closest match to your internet connection. Try selecting a lower setting if you're getting huge lag."
	NetSpeeds(0)="Modem (28.8K - 56K)"
	NetSpeeds(1)="ISDN"
	NetSpeeds(2)="Cable, xDSL"
	NetSpeeds(3)="LAN"
```

Need add more options, and make better descriptions to exists items. Possible with state exactly numbers, since numbers never lie and never outdate. 20 KB/s in 2000 year is same as 20 KB/s now.

**Comment by SeriousBarbie** [comment]
Here in Germany ISDN even doesn't exist any more.

**Comment by SeriousBuggie** [comment]
No. Need clear explanation. I understand when game out nobody think about years which it survive. And how far techs can goes ahead. So on time for release such stuff is acceptable.

Now we know game will be live long. And must take this in account.

"LTE" mean nothing. Today is one speed, Tomorrow it be cosmic tech which provide 100 GB/s and called as `9G LTE`.
But players still select it, because they know nothing.

Not need goes to same trap twice. You never know which tech be tomorrow and how it be named.
Need show real numbers and maybe in brackets some info for understand. But need be careful which this info.

Also numbers must be labeled as B/s, not as nameless 20000.

**Comment by Calinou** [comment]
Do we even need those options in the first place? Today's mobile connections can handle the "LAN" setting just fine in my experience. Using slower values is just a net detriment in 2025.

The Network tab could even be removed entirely perhaps, unless there are other worthwhile options to add there.
